crates/holochain/src/lib.rs

```diff
@@ -58,3 +58,12 @@ pub mod prelude {
     #[cfg(feature = "test_utils")]
     pub use holochain_types::inline_zome::*;
 }
+
+
+#[cfg(all(feature = "wasmer_sys", feature = "wasmer_wamr"))]
+compile_error!(
+    "feature \"wasmer_sys\" and feature \"wasmer_wamr\" cannot be enabled at the same time"
+);
+
+#[cfg(all(not(feature = "wasmer_sys"), not(feature = "wasmer_wamr"),))]
+compile_error!("One of: `wasmer_sys`, `wasmer_wamr` features must be enabled. Please, pick one.");
```

Comment:
I saw the wasmer crates use this pattern for specifying mutually exclusive and required feature flags. I thought it seemed nice. 

I don't see it used elsewhere in the holochain crates so not sure if this is a time to start using it.